404.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html PUBLIC 
+<html lang="pt-br">
+<head>
+<title>Multiversus blog - Home</title>
+<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
+<link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+<div id="wrapper">
+  <div id="top">
+    <img src="images/Logo.png" width="760" height="217" alt="" id="main-image" />
+  </div>
+  <div id="navigation">
+    <ul>
+      <li class="active"><a href="home.html">Home</a></li>
+      <li><a href="sobremim.html">Sobre mim</a></li>
+      <li><a href="artigos.html">Artigos</a></li>
+    </ul>
+  </div>
+  <img src="images/404.png" width="760" height="317" alt="" id="main-image" />
+  <div id="content" class="wrapper">
+    <div id="main-content">
```

Comment:
daria pra usar a tag de main, ai fica menos "confuso" pra quem for ler, mas funciona do mesmo jeito